### PR TITLE
A new feature that detects files with mixed line ending

### PR DESCRIPTION
**Pull request by realmarv**
Detecting files with mixed line ending.

### CODE REVIEW

File: mixedLineEnding.fsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+open System.IO
+open System
+
+let mixedLineEnding(fileInfo: FileInfo) =
```
Comment (knocte) [inline mixedLineEnding.fsx:4]: @realmarv we use PascalCase for functions

Comment (realmarv) [reply mixedLineEnding.fsx]: Done

File: mixedLineEnding.fsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 open System.IO
 open System
 
-let mixedLineEnding(fileInfo: FileInfo) = 
+let MixedLineEnding(fileInfo: FileInfo) =
```
Comment (knocte) [inline mixedLineEnding.fsx:4]: @realmarv oh, and it should be plural, right? because I don't think it's checking for 1 line ending

Comment (realmarv) [reply mixedLineEnding.fsx]: Is using ReadToEnd a bad practice here as well? I guess it doesn't include line endings in each line when I use ReadLines(), right?

Comment (knocte) [reply mixedLineEnding.fsx]: yes, there's no reason to use ReadToLine if it can work with ReadLines()

Comment (knocte) [reply mixedLineEnding.fsx]: > I guess it doesn't include line endings in each line when I use ReadLines(), right?

Oh good point, maybe we cannot use ReadLines() :'(

**Pull request merged**